aten/src/ATen/core/DimVector.h

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+#include <ATen/core/SmallVector.h>
+#include <stdint.h>
+
+namespace at {
+
+/// A container for sizes or strides
+using DimVector = SmallVector<int64_t, 5>;
```

Comment:
Wondering idly: maybe the max here should be 4 rather than 5, based on the fact that Caffe2 historically defined

```
  SetDims(const TIndex d0, const TIndex d1, const TIndex d2, const TIndex d3) {
```

but not the 5-ary version.

Reply:
Maybe other people know better.

Reply:
Yeah c2 is probably using 4 more often.
this is defined earlier, I'm going to change it to DimVector in aten which is 5 elements.
Should we change or use the one in aten to be consistent(or change at::DimVector to be 4 elements?)?

Reply:
keeping it 5 since it's default for 3D data.